docs/using/top-bar.md

```diff
@@ -4,26 +4,24 @@
 
 When an app is running in Sandstorm, Sandstorm provides navigation elements at the top of the screen.
 
-You can see an example here:
-
-* [https://sandstorm.io/vote](https://sandstorm.io/vote) -- this redirects to a URL on `alpha.sandstorm.io`.
+You can see an example by opening an [EtherCalc demo grain](https://demo.sandstorm.io/appdemo/a0n6hwm32zjsrzes8gnjg734dh6jwt7x83xdgytspe761pe2asw0).
```

Comment:
I'd prefer s/opening an/launching a new/ to avoid people thinking that there is a shared grain they are going to get access to via this link.

Usually in my experience you don't have strong opinions about this kind of change, so I might submit a PR on top of this one and self-merge it with that change.